Libraries/acdriver/Sources/Compile/ImageSet.cpp

```diff
@@ -59,6 +59,16 @@ GenerateIdentifier(void) {
     return last;
 }
 
+static car::Rendition::Data::Format
+ExtraImageTypeToDataFormat(Output::ExtraImageType type) {
+    switch (type) {
+        case Output::ExtraImageType::WEBP:
+            return car::Rendition::Data::Format::WEBP;
+    }
+
+    abort();
```

Comment:
Do you need abort?  Compilation should catch incomplete switch cases -- yours looks correctly exhausted.

Reply:
I need it for MinGW (or windows build breaks)...